### DISCUSSION TITLE
Is it possible to separate different artists that have the same name ?

### DISCUSSION THREAD

**Discussion by rombat** [navidrome/navidrome#3144] [Q&A]
I have a few artists that have the same name, and that are considered in navidrome like a single one
Is there a way to separate them ?
Some of them have different MUSICBRAINZ ARTIST ID tag, but some tracks just don't have this tag, so I don't know how it would be ideal (if possible) to separate them all (let alone when it's a multiple artists track >_>)

**Comment by metalheim** [comment]
See this kinda related issue #2004 
It will be possible to use the musicbrainz id to distinguish between same-name artists after the BFR #2709
